Formula/gitbackup.rb

```diff
@@ -0,0 +1,18 @@
+class Gitbackup < Formula
+  desc "Tool to backup your Bitbucket, GitHub and GitLab repositories"
+  homepage "https://github.com/amitsaha/gitbackup"
+  url "https://github.com/amitsaha/gitbackup/archive/refs/tags/0.6.tar.gz"
+  sha256 "66edd50945a689abcd07aad5047317dc17cb17ec5e813f63c3520394696a6fcb"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    assert_match(/Please specify the git service type/, shell_output("#{bin}/gitbackup 2>&1", 1))
```

Comment:
```suggestion
    assert_match "Please specify the git service type", shell_output("#{bin}/gitbackup 2>&1", 1)
```
You don't need a `Regexp` here.